app/containers/ViewCompany/actions.js

```diff
@@ -9,10 +9,17 @@ export const getCompany = id => ({
   id,
 });
 
-export const getCompanySuccess = company => ({
-  type: GET_COMPANY_SUCCESS,
-  company,
-});
+export const getCompanySuccess = fetchedCompany => {
+  const { address, ...company } = fetchedCompany;
+  company.address = `${address.streetName} ${address.streetNumber}, ap. ${
```

Comment:
Don't do this here. Let the component that displays the address to decide how it looks like.